cumulus/parachains/integration-tests/emulated/chains/parachains/people/people-rococo/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "people-rococo-emulated-chain"
+version = "0.0.0"
```

```suggestion
version = "0.1.0"
```

I don't see why this should be different from the below version, even though it's unpublished.